### DISCUSSION TITLE
在模型本地运行的时候报了这个错误

### DISCUSSION THREAD

**Discussion by Zengfanxu1111** [zai-org/ChatGLM3#583] [Q&A] [answered]
Symbol cudaLaunchKernel not found in C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common\cudart64_65.dll

CUDA11.7
RTX 2060

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
环境太低了，按照要求配环境
